src/scripts/main.js

```diff
@@ -741,7 +741,16 @@
 
         // Perform validations and stage corrections.
         fixLGLocations(message, user_pre, user_post, hunt);
+        const temp_message_post = message; // For transition check
```

Comment:
This is a shallow reference; any changes to the properties of `message` are also made to `temp_message_post`.

Reply:
thank you. Javascript never ceases to annoy me :)